translations/ui-users/en.json

```diff
@@ -697,33 +697,34 @@
   "transfers.singular": "Transfer account",
   "transfers.nodata": "There are no Fee/fine: Transfer accounts",
 
+  "reports.claimsReturned.label": "Claims returned report (CSV)",
```

Comment:
Please pay attention that the menu item should be named "Claim**ed** returned report (CSV)".

Reply:
corrected.